### ISSUE TITLE
No single pointer is provided to drag the "columns headers" to the required position in the "Keyboard shortcuts" Table: A11y_Visual Studio Code Client_Customizations and settings_Keyboard Shortcuts_Dragging movements. 

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#221554]
### GIT Hub Tags:
#A11yWCAG2.2; #A11yTCS; #A11ySev2; #Visual Studio Code Client; #BM_Visual Studio Code Client_MAC_JULY2024; #DesktopApp; #WCAG2.5.7; #FTP; #A11yMAS; #Dragging Movements; #MAC; #Device;

### Environment and OS details:
MAC Book Pro 
MAC OS Sonoma Version 14.5
Visual Studio code Insider version 1.92.0

### Reproduction Steps:

1. Open Visual Studio Code Insiders editor
2. Tab Navigate to setting icon.
3. Select "Keyboard shortcuts options" and press enter or Press keyboard shortcut as command K, Command S 
4. Keyboard shortcut table gets opened.
5. Observe that No single pointer is provided to drag the "columns headers" to the required position in the "Keyboard shortcuts" Table.

### Actual behavior:
No single pointer is provided to drag the "columns headers" to the required position in the "Keyboard shortcuts" Table.

### Expected behavior:
There should be single pointer provided to drag the "columns headers" to the required position in the "Keyboard shortcuts" Table.

### User Impact:
People with motor impairments who cannot operate content because the only way to actuate a function is by dragging a target element from its initial position to some other position will face difficulty in accessing the application.

### Attachments:

https://github.com/user-attachments/assets/ae697a5e-89ea-4831-8c96-beff3756824f

**Comment by PoojaNamde** [comment]
#Rev:poojanamde;

**Comment by meganrogge** [comment]
@ulugbekna do you want help with this? We should fix it by September

**Comment by meganrogge** [comment]
working on this!

**Comment by rzhao271** [comment]
Assigned a keybinding to list.resizeColumn and was able to resize the columns of the Keyboard Shortcuts table afterwards.
